makefile

```diff
@@ -0,0 +1,11 @@
+all: generate qkchashmake
+
+generate:
+	python -m grpc_tools.protoc --proto_path=./quarkchain/cluster --python_out=./quarkchain/generated --grpc_python_out=./quarkchain/generated grpc_client.proto
+
+qkchashmake:
```

Comment:
nit: just `qkchash` to be succinct?

Reply:
it will have name conflicts with the directory of qkchash. I will use another succinct name.

Reply:
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

```
.PHONY: qkchash
all: proto qkchash

proto: quarkchain/cluster/grpc_client.proto
	python -m grpc_tools.protoc --proto_path=quarkchain/cluster --python_out=quarkchain/generated --grpc_python_out=quarkchain/generated grpc_client.proto

qkchash:
	$(MAKE) -C ./qkchash

clean:
	rm -f quarkchain/generated/*.py
	$(MAKE) -C ./qkchash clean

```